src/App.tsx

```diff
@@ -21,7 +21,7 @@ export default function App() {
   // vvvvvvvvvvvvvvvvv
 
   let yourBlock: BlockModel = {
-    type: "%NAME%",
+    type: "PhotoAlbum",
```

Comment:
I've been meaning to write a git commit hook to return this to `%NAME%` when you submit it, but until then, could you replace the block name with `%NAME%` here so that the seam_magic script runs for the next person?